<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>669</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>820</width>
    <height>649</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>830</width>
    <height>669</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Driver Loader 1.0</string>
  </property>
  <property name="windowIcon">
   <iconset resource="img.qrc">
    <normaloff>:/img/d.jpg</normaloff>:/img/d.jpg</iconset>
  </property>
  <property name="toolTip">
   <string>Driver Loader 1.0</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="maximumSize">
    <size>
     <width>865</width>
     <height>650</height>
    </size>
   </property>
   <widget class="QGroupBox" name="CrtlBGrpBox">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>250</y>
      <width>261</width>
      <height>161</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>Control B</string>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid  #4CAF50;
border-radius: 25px;</string>
    </property>
    <property name="title">
     <string>Control B</string>
    </property>
    <widget class="QPushButton" name="stopBtn">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>100</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Stop a driver as a service</string>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Stop Service</string>
     </property>
    </widget>
    <widget class="QPushButton" name="RegBtn">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Register a driver as a service</string>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Register Service</string>
     </property>
    </widget>
    <widget class="QPushButton" name="UnregBtn">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Unregister a driver as a service</string>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Unregister Service</string>
     </property>
    </widget>
    <widget class="QPushButton" name="startBtn">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>start a driver as a service</string>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Start Service</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="CtrlAGrpBox">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>30</y>
      <width>261</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>Control A</string>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid  #4CAF50;
border-radius: 25px;</string>
    </property>
    <property name="title">
     <string>Control A</string>
    </property>
    <widget class="QPushButton" name="ProcessBtn">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>Display running processes and there IDs</string>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Process</string>
     </property>
    </widget>
    <widget class="QPushButton" name="BrowseBtn">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>Get driver from path</string>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ClearBtn">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>Clear form</string>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ServicesBtn">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>110</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>Display driver activity</string>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Services</string>
     </property>
    </widget>
   </widget>
   <widget class="QPlainTextEdit" name="PlainTxtEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>470</y>
      <width>261</width>
      <height>121</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>About program</string>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid #4CAF50</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>430</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>About</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="FromGrpBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>461</width>
      <height>571</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>Driver details form</string>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>Form</string>
    </property>
    <widget class="QComboBox" name="ErrorLevelComboBox">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>400</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Set error level</string>
     </property>
    </widget>
    <widget class="QComboBox" name="StartComboBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>480</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Set driver start procedure</string>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="currentText">
      <string/>
     </property>
     <property name="insertPolicy">
      <enum>QComboBox::NoInsert</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>480</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Service Start:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="ServiceComboBox">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>400</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Type of driver</string>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>400</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Service Type:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>400</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Error:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="DisplayNameLineEdit">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>340</y>
       <width>281</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:2px  solid #4CAF50; 
border-radius: 8px;
 </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>340</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Display Name:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ServiceNameLineEdit">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>280</y>
       <width>281</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:2px  solid #4CAF50; 
border-radius: 8px;
 </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>280</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Service Name:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>220</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Driver Time:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="DriverTimeLineEdit">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>220</y>
       <width>281</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:2px  solid #4CAF50; 
border-radius: 8px;
 </string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="DriverSizeLineEdit">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>160</y>
       <width>281</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:2px  solid #4CAF50; 
border-radius: 8px;
 </string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>160</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Driver Size:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Driver Version:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="DriverVersionLineEdit">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>100</y>
       <width>281</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:2px  solid #4CAF50; 
border-radius: 8px;
 </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="DriverPathLineEdit">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>281</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:2px  solid #4CAF50; 
border-radius: 8px;
 
</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Driver Path:</string>
     </property>
    </widget>
   </widget>
   <zorder>FromGrpBox</zorder>
   <zorder>CtrlAGrpBox</zorder>
   <zorder>CrtlBGrpBox</zorder>
   <zorder>PlainTxtEdit</zorder>
   <zorder>label_10</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>830</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
